app/src/main/java/com/tvtracker/dao/IMediaItemDAO.kt

```diff
@@ -7,7 +7,14 @@ import retrofit2.http.Query
 
 interface IMediaItemDAO {
 
+    /**
+     * Calls data from data transfer object.
+     */
     @GET("/")
-    fun searchIMDB(@Query("s") searchTxt: String, @Query("type") type: String, @Query("page") page: Int): Call<ImdbResponse>
+    fun searchIMDB(
+        @Query("s") searchTxt: String,
```

Comment:
Restructured dao to proper coding conventions for readability.
https://developer.android.com/kotlin/style-guide#formatting